src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    private String color;
+    private double area;
```

Comment:
The `area` variable is defined at the class level, but it might be more appropriate to calculate the area dynamically in the method that requires it, rather than storing it as a class-level variable.